operator/redisfailover/service/constants.go

```diff
@@ -40,4 +40,7 @@ const (
 	redisGroupName         = "mymaster"
 	appLabel               = "redis-failover"
 	hostnameTopologyKey    = "kubernetes.io/hostname"
+	redisRoleLabelName     = "RedisRole"
```

Comment:
Since `RedisRole` is quite redundant for the components of operator I would prefer it was `role`

Reply:
It's OK.  

Thanks